### PR TITLE
Blog post: Alt text event summary

### PR DESCRIPTION
**Pull request by isabela-pf**
**Please note:** this PR is not in draft mode because it is open for review, but it is not yet ready for merging. It is still missing post-event links, like the YouTube recording, that don't exist yet. I'll edit this note when it's ready to be merged.**

## What this is

This post summarizes the [image description event](https://twitter.com/scientific_py/status/1523733881651834880) held for the blog earlier in May. It aims to highlight what was done while providing resources for anyone reading to follow up.

## What needs to be done

- [x] Add recording link
- [x] Add contributing PR link
- [x] Add final number of image descriptions added and posts impacted
- [ ] Someone with relevant knowledge needs to check that the post metadata is all correct
- [x] Add the group photo we took to line 40. 
- [x] @MarsBarLee needs to review
- [ ] Update post date when we have an accurate estimate

Thanks in advance for the review!

### CODE REVIEW

**Review by tupui — COMMENTED** [review]
Thank you for the PR 🚀 For the image, can you run it through `pngquant` or something similar? Locally I can half the size of the image. See https://blog.scientific-python.org/submitting/#images

Here it is for conveniency
alt-text-header-fs8

**Comment by isabela-pf** [comment]
@tupui thanks for pointing that out; it's 100% my fault I missed that step. Do I need to do anything about the prior commit with the larger image?

**Comment by tupui** [comment]
Thanks for the update @isabela-pf. No need to do anything about the previous commit since we are squashing all PRs on this repo. But good thinking 👍🚀

File: content/posts/scientific-python/alt-text-workshop-summary/index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+title: "Team up! Alt text and cross-project community"
+date: 2022-05-19
+draft: false
+tags: ["Scientific-Python", "event", "acessibility"]
+description: "Learn more about alt text on the blog and how you can contribute."
+displayInList: true
+author: ["Mars Lee", "Isabela Presedo-Floyd"]
+resources:
+  - name: featuredImage
+    src: "alt-text-header.png"
```
Comment (tupui) [inline content/posts/scientific-python/alt-text-workshop-summary/index.md:11]: @jarrodmillman do we have a way to put alt-text there?

File: content/posts/scientific-python/alt-text-workshop-summary/index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+the blog, the work to add or imporve alt text on Scientific Python's blog, 
+website, and documentation is ongoing. You can continue these efforts by
+
+- Contributing as an individual! There's no reason you can't write some image descriptions for your favorite blog post to make it even better. (👀 This is a great excuse to find a new favorite blog post.)
+- Contributing as a group! Collect your friends, co-workers, neighbors, local book club attendees, or others to team up and write alt text. (Help for duplicating the process of our group PR can be found [on the draft event documentation](https://github.com/isabela-pf/a11y-events/tree/main/workshop-resources/alt-text).)
+- Staying in touch for annoucements of future contributing events on the [Scientific Python Twitter](https://twitter.com/scientific_py).
+
+---
+
+*This image description event has been run across multiple open-source projects in the scientific computing ecosystem. [Reach out via issue](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/issues/new/choose) if you are interested in running a similar event on a project you are part of.* ❤️
```
Comment (MarsBarLee) [inline content/posts/scientific-python/alt-text-workshop-summary/index.md:71]: This link leads to the Jupyter CZI accessibility grant. Maybe a better place would be to link to [a11y-events](https://github.com/isabela-pf/a11y-events/issues/new) or the [new landing page for the workshops](https://github.com/alt-text-task-force)?

Comment (MarsBarLee) [inline content/posts/scientific-python/alt-text-workshop-summary/index.md:71]:
```suggestion
*This image description event has been run across multiple open-source projects in the scientific computing ecosystem. [Reach out via issue](https://github.com/isabela-pf/a11y-events/issues/new) if you are interested in running a similar event on a project you are part of.* ❤️
```

Comment (MarsBarLee) [inline content/posts/scientific-python/alt-text-workshop-summary/index.md:71]:
```suggestion
*This image description event has been run across multiple open-source projects in the scientific computing ecosystem. [Reach out via issue](https://github.com/alt-text-task-force) if you are interested in running a similar event on a project you are part of.* ❤️
```

**Comment by MarsBarLee** [comment]
This is a really great blogpost, thank you Isabela for writing it! I also appreciate the linking out to resources, that this is a common accessibility problem and other smaller things, like the agenda and previous social media promotion. It really puts everyone's work in context- that it wasn't just a 1 hour event, it's more than that :)

**Comment by juanis2112** [comment]
HI @MarsBarLee, @tupui and @isabela-pf, I finally got the recording from Jarrod and did some editing to upload it to YouTube. Here is the link https://youtu.be/Zn-zyU2lS0k for you to include in the blogpost. It is "unlisted" which means only people with the link can see it but once you and @stefanv approve, it will go public. 

I made a thumbnail using the promo picture you sent me but if you want something different please let me know. I can change it. Also, I cropped most of the "silent" parts because there were only your faces silent for some minutes while people were working but not much to see and I also stopped at the end of the workshop and didn't include the discussion at the end to avoid making it too long. That's why the recording is only 47 minutes. I also included some intro with the name and date and some descriptions for the video which I took from the Twitter post we did. (But again I can change those too).

Finally, I got the images too for you to include in the blogpost. You should be able to download them from here and choose the one you like the most:
screenshot1
screenshot2
screenshot3

Please let me know if there is something else you need. Once the blogpost is merged I'll post it on our twitter so we can get more people to read it!

**Comment by isabela-pf** [comment]
Wonderful! Thank you @juanis2112, I've made the changes. Please let me know if anything else needs updating or editing!

**Comment by tupui** [comment]
I took the liberty to compress the image and fix the linters.

**Comment by isabela-pf** [comment]
Thank you! Sorry about that, perhaps I didn't crush it properly.

**Review by tupui — APPROVED** [review]
Thank you all 👏 let's get this in and if there is anything to modify we can always fix it later.

**Pull request merged**